outOfSyncNoti.sh

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
-# A simple shell script to check if node is out of sync with ethereum testnet
-# If node is out of sync, user will be nptofied by OpsGenie
-# Using crontab command: `*/10 * * * * /home/someuser/tmp/` to run this script every 10 mins.
+# A simple shell script to check if node is out of sync with ethereum
+# If node is out of sync, user will be notified by OpsGenie
+# Using crontab command: `*/10 * * * * /home/foo/outOfSyncNoti.sh 'rinkeby' 'opsgeniekeyhere' >> /home/foo/nodeSync.log 2>&1` to run this script every 10 mins.
+
+network=$1
+port=$2
```

Comment:
some comments explaining the params here would be nice:
- what is port? port of opsGenie API, port of geth API, port this script is listening on?
- what is treshold? number of blocks behind, unit of time it has been going on? what unit? minutes, seconds?